documents/xaml-suggested-actions/xaml-designer-suggested-actions-extensibility-tokens.md

```diff
@@ -0,0 +1,32 @@
+## Suggested Actions Tokens
+
+Each action group and action could have assigned token of type `ActionToken`. Using this token, action group or action could be referenced and modified. Example:
```

Comment:
"could have assigned token" -> "could have an assigned token"
" Using this token, action group or action could" ->  "Using this token, the corresponding action group or action can"